### PR TITLE
Update grass_setup.rst

### PR DESCRIPTION
**Pull request by janneke-qgis**
Trying again :-). In Windows (don't know about the others) you need to start QGIS 3.4 with Grass 7.4 otherwise you will not be able to install/activate the grass plugin

### CODE REVIEW

File: source/docs/training_manual/grass/grass_setup.rst [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 Using GRASS in QGIS requires you to think of the interface in a slightly
 different way. Remember that you're not working in QGIS directly, but working
-in GRASS *via* QGIS.
+in GRASS *via* QGIS. If you are working in Windows, make sure that you have started QGIS Desktop 3.4 with Grass 7.4.
```
Comment (havatv) [inline source/docs/training_manual/grass/grass_setup.rst]:
```suggestion
in GRASS *via* QGIS.

.. note:: If you are working in Windows, make sure that you have QGIS Desktop
with Grass support.
```

Comment (havatv) [reply source/docs/training_manual/grass/grass_setup.rst]: I suggest that a note is used for this, and I think that we should avoid mentioning the specific QGIS and GRASS versions.

Comment (janneke-qgis) [reply source/docs/training_manual/grass/grass_setup.rst]: Better. Would it be self-evident how to start QGIS with grass support?

Comment (havatv) [reply source/docs/training_manual/grass/grass_setup.rst]: I guess that should be covered by the user manual.  If you can find where it is described, you could provide a "reference".

Comment (DelazJ) [reply source/docs/training_manual/grass/grass_setup.rst]: We have https://docs.qgis.org/testing/en/docs/user_manual/grass_integration/grass_integration.html#grass-gis-integration but at first glance there's no clear instructions on this and i also wonder how duplicate contents are with the training manual. 
But is it a Windows-only prerequisite? Anyone with grass experience on other platform?

Comment (DelazJ) [reply source/docs/training_manual/grass/grass_setup.rst]: We have https://docs.qgis.org/testing/en/docs/user_manual/grass_integration/grass_integration.html#grass-gis-integration but at first glance there's no clear instructions on this and i also wonder how duplicate contents are with the training manual. 
But is it a Windows-only prerequisite? Anyone with grass experience on other platform?

Comment (DelazJ) [reply source/docs/training_manual/grass/grass_setup.rst]: We have https://docs.qgis.org/testing/en/docs/user_manual/grass_integration/grass_integration.html#grass-gis-integration but at first glance there's no clear instructions on this and i also wonder how duplicate contents are with the training manual. 
But is it a Windows-only prerequisite? Anyone with grass experience on other platform?

Comment (ghtmtt) [reply source/docs/training_manual/grass/grass_setup.rst]: I just have experience with GRASS + QGIS on QGIS self compiled on Linux where you need to export some environmental variables... I'm pretty sure if you are on Linux system with QGIS installed with standard packages (AKA repository) then you don't have any trouble.

Let me know if you want to describe also this option,but I think it's a little bit too much ;)

Comment (DelazJ) [reply source/docs/training_manual/grass/grass_setup.rst]: Official instructions in installing QGIS on Linux require to install qgis_plugin_grass so it should be good for those platforms and osgeo4w provides it (by default?). But maybe we could simply skip the "If you are working in Windows, " part and point to the aforementioned grass chapter and have someone add the information about how to get grass support there.

**Comment by DelazJ** [comment]
Up...

**Comment by ghtmtt** [comment]
@DelazJ +1, I agree.

**Comment by ghtmtt** [comment]
What about committing the last suggestion and merge this? I don't see any reason to keep it in the cue..

ping @DelazJ?

File: source/docs/training_manual/grass/grass_setup.rst [hunk 2]
```diff
@@ -3,7 +3,7 @@
 
 Using GRASS in QGIS requires you to think of the interface in a slightly
 different way. Remember that you're not working in QGIS directly, but working
```
Comment (DelazJ) [inline source/docs/training_manual/grass/grass_setup.rst]: Do I misunderstand something or is the logic inverted here?
```suggestion
different way. Remember that you're not working in GRASS directly, but working
```

File: source/docs/training_manual/grass/grass_setup.rst (hunk 1 above)
Comment (DelazJ) [inline source/docs/training_manual/grass/grass_setup.rst]:
```suggestion
in GRASS *via* QGIS. Hence, make sure you have installed QGIS Desktop with Grass support.
```

**Comment by ghtmtt** [comment]
@DelazJ could you force the commits of your review? This PR is almost 2 months old and it seems we agree to merge (and backport) it

**Comment by DelazJ** [comment]
Thanks @janneke-qgis

**Pull request merged**